### DISCUSSION TITLE
CDISC standard agents

### DISCUSSION THREAD

**Discussion by nicholas-masel** [rstudio/pointblank#436] [Ideas]
Industry standards should allow us to create standard agents. Are you aware of industry collaborations to build standard agents? (thinking CDISC, eDC to ADaM)

**Comment by rich-iannone** [comment]
I am not aware of any, so the opportunity is there to create a package of this sort that uses pointblank to help generate the custom validation functions. 

Let me know if this sounds interesting as a project you might take on. I can assist in any capacity necessary.

**Reply by nicholas-masel** [reply]
Thanks Rich!  Let me float this around and reach out the pharmaverse folks as well and see if there is interest here.

**Reply by nicholas-masel** [reply]
I spoke with @mstackhouse, who is on the [pharmaverse ](https://pharmaverse.org/) council, and he mentioned [CORE](https://cosa.cdisc.org/directory/core) is covering this, just in python.

There is also [sdtmchecks](https://pharmaverse.org/e2eclinical/sdtm/) for R, that will be coming to the pharmaverse by the end of this year, which will will cover some data validation checks.  This doesn't use pointblank, but I'm thinking of an exercise to convert a few of these sdtmchecks to utilize pointblank and see what additional benefits this package would get for "free" if it refactored to pointblank, and see if there is any interest by this team and if they want some additional support.

**Reply by rich-iannone** [reply]
This all sounds very promising! If you do work on that pilot project, please let me know if I could be of any assistance.
